examples/community/stable_diffusion_tensorrt_txt2img.py

```diff
@@ -0,0 +1,533 @@
+#
+# Copyright 2023 The HuggingFace Inc. team.
+# SPDX-FileCopyrightText: Copyright (c) 1993-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import numpy as np
+from typing import List, Optional, Union
+
+import torch
+import tensorrt as trt
+from polygraphy import cuda
+from transformers import CLIPFeatureExtractor, CLIPTextModel, CLIPTokenizer
+
+from tensorrt_utils import TRT_LOGGER, build_engines, device_view, runEngine, Optimizer, BaseModel
```

Comment:
could we move `tensorrt_utils.py` into the same file as this one? :-) Otherwise I think we cannot run the community pipeline out of the box